File: go/benchmarks/utils/connectionsettings.go
package utils

type ConnectionSettings struct {
	Host   string
	Port   int
	UseSsl bool
}

func NewConnectionSettings(host string, port int, useSsl bool) *ConnectionSettings {
	return &ConnectionSettings{
		Host:   host,
		Port:   port,
		UseSsl: useSsl,
	}
}

File: java/benchmarks/src/main/java/javababushka/benchmarks/utils/ConnectionSettings.java
package javababushka.benchmarks.utils;

/** Redis-client settings */
public class ConnectionSettings {
  public String host;
  public int port;
  public boolean useSsl;

  public ConnectionSettings(String host, int port, boolean useSsl) {
    this.host = host;
    this.port = port;
    this.useSsl = useSsl;
  }
}
